spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/SparkTransformConverter.java

```diff
@@ -39,6 +47,17 @@
  */
 public class SparkTransformConverter {
 
+  /**
+   * supportsBucketPartition is used to indicate whether a dataSource supports bucket partition. For
+   * example, iceberg supports bucket partition. We use true to indicate that bucket partition are
```

Comment:
iceberg -> Iceberg

Reply:
replace  `We use true to indicate xxx ` with`If supportsBucketPartition  is ture, BucketTransform is transfromed to partition, and SortedBucketTransform is not supported. If false,  BucketTransform and SortedBucketTransform is transformed to Distribution and SortOrder.` ?

Reply:
ok.

Reply:
updated